bigml/bigmlconnection.py

```diff
@@ -783,7 +783,7 @@ def error_message(self, resource, resource_type='resource', method=None):
                         resource_type, resource_type))
                 return error
             if code == HTTP_UNAUTHORIZED:
-                error += u'\nDouble-check your credentials, please.'
+                error += u'\nDouble-check your credentials and the general domain your account is registered with, please.'
```

Comment:
Thanks for the feedback! We try to comply with the 79 column limit to ensure code readability. Could you please adapt your change to this requirement? You could also add the information about the domain it is currently pointing to. Maybe something like:

```python
                error += u'\nDouble-check your credentials and the general'
                         u' domain your account is registered with, please'
                         u' (currently using %s).' % self.general_domain
```